common/lib/xmodule/xmodule/backcompat_module.py

```diff
@@ -60,6 +60,8 @@ def from_xml(cls, xml_data, system, id_generator):
 
 
 class SemanticSectionDescriptor(XModuleDescriptor):
+    resources_dir = None
```

Comment:
It makes a lot of sense, but then it would break all third-party XModules that *do* use the resource loading API. :/